const notFoundError = (res) => {
    return res
        .status(404)
        .send('Esse dado n√£o foi encontrado no banco de dados.');
};

module.exports = {
    notFoundError,
};
